kong/templates/nginx_kong.lua

```diff
@@ -76,7 +81,13 @@ server {
     error_page 400 404 405 408 411 412 413 414 417 494 /kong_error_handler;
     error_page 500 502 503 504                     /kong_error_handler;
 
-    access_log ${{PROXY_ACCESS_LOG}};
+    set $kong_request_id $request_id;
```

Comment:
Why don't we just use `$request_id`?

And the `set` directive is from the `ngx_http_rewrite_module`, it will run on the `rewrite` phase, which means that any log before the `rewrite` phase has no request id, it this by design?